src/test/java/org/apache/commons/lang3/function/ConsumersTest.java

```diff
@@ -46,4 +50,15 @@ public void testNop() {
         Consumers.nop().accept("");
     }
 
+    /**
+     * Tests {@link Consumers#acceptIfNotNull(Object, Consumer)}.
+     */
+    @Test
+    public void testAcceptIfNotNull() {
+        StringBuilder builder = new StringBuilder("jay");
+        Consumers.acceptIfNotNull(builder, sb -> sb.append("-bae"));
```

Comment:
Don't use your name as test data, just use data like foo and bar like you did in the other test.